deployment/7_deploy_2.5.js

```diff
@@ -203,6 +205,16 @@ const main = async () => {
   console.log(`Deployed UniswapV2Filter at ${UniswapV2FilterWrapper.address}`);
   await DappRegistryWrapper.addDapp(0, config.defi.uniswap.unizap, UniswapV2FilterWrapper.address);
 
+  // Aave V1
+  console.log("Deploying AaveV1");
```

Comment:
I think we should also authorise the [LendingPoolCore] (https://etherscan.io/address/0x3dfd23A6c5E8BbcFc9581d2E864a68feb6a076d3) contract (used as the target of an ERC20 approval prior to depositing the ERC20 into the lending pool) with our OnlyApproveFilter.

Reply:
And we could also add that "ERC20 deposit" use case to our tests as well